go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/csi-addons/kubernetes-csi-addons
 
-go 1.22.0
+go 1.22.5
```

Comment:
i dont think this need to change, can you please check, changing/adding toolchain to 22.6 is enough

Reply:
it was modified by `go mod tidy
let me re confirm it.

Reply:
@Madhu-1, Run Golang test fails
```
go: updates to go.mod needed; to update it:
	go mod tidy
```

Reply:
got it, Sorry i missed that, can you please update this as well?